lib/cloudWatch.js

```diff
@@ -23,18 +24,58 @@ export default class CloudWatch {
     this.namespace = process.env.SERVICE_NAME || 'coinboss';
   }
 
-  putPriceMetric(args) {
-    return this.cloudwatch.putMetricData({
+  putMetricData(args) {
+    const data = {
       MetricData: [
+       {
+         MetricName: args.metricName,
+         Timestamp: moment().utc().format(),
```

Comment:
Well, I ran into issues with CloudWatch and the required time format. This way I was certain the format is in UTC. if `new Date()` works the same I should revert.